docs/examples/video_decoding/README.md

```diff
@@ -0,0 +1,42 @@
+# Decoding video in DALI backend
+
+## Splitting sequences into subsequences
+Often the video model we want to run inference on expects sequences of some fixed lentght. In such case we may want to split the decoded video
+into a batch of sub-sequences of that given length. We can achieve that utilizing the `fn.reshape` operator from DALI together with a model parameter
+`"split_along_outer_axis"` set in configuration file. Lets see an example of using this combination.
+
+```python
+FRAMES_PER_SEQUENCE = 5
+OUT_WIDTH = 300
+OUT_HEIGHT = 300
+
+@autoserialize
+@dali.pipeline_def(batch_size=256, num_threads=4, device_id=0,
+                   output_dtype=dali.types.UINT8, output_ndim=5)
+def pipeline():
+  vid = fn.external_source(device='cpu', name='INPUT', ndim=1, dtype=dali.types.UINT8)
+  seq = fn.experimental.decoders.video(vid, device='mixed')
+  seq = fn.resize(seq, resize_x=OUT_WIDTH, resize_y=OUT_HEIGHT)
+
+  seq = fn.pad(seq, axis_names='F', align=FRAMES_PER_SEQUENCE)
```

Comment:
Resize is kind of necessary, because we need to know frame size to perform the reshape.
It wouldn't be needed if we knew that all of our files have some known resolution